source/_integrations/kef.markdown

```diff
@@ -0,0 +1,80 @@
+---
+title: "KEF Speakers"
+description: "Instructions on how to integrate KEF Speakers into Home Assistant."
+logo: kef.png
+ha_category:
+  - Media Player
+ha_iot_class: Local Polling
+ha_release: 0.103
+---
+
+The `kef` platform allows you to control the [KEF LS50 Wireless](https://international.kef.com/products/ls50-wireless) and [KEF LSX](https://international.kef.com/products/lsx) speakers from Home Assistant.
+
+Supported devices:
+
+- KEF LS50 Wireless
+- KEF LSX (untested)
+
+To add KEF Speakers to your installation, add the following to your `configuration.yaml` file:
+
+## Configuration
+
+```yaml
+# Example configuration.yaml entry
+- platform: kef
+  host: IP_ADDRESS
+  type: LS50
+```
+
+{% configuration %}
```

Comment:
I'm missing the `port` configuration variable

Reply:
Thanks @klaasnicolaas for your careful reading! I've added it in dc1d1d2.